ert_shared/main.py

```diff
@@ -413,10 +414,10 @@ def main():
     if args.verbose:
         logger.setLevel("DEBUG")
     FeatureToggling.update_from_args(args)
-    logger.info("Running ert with {}".format(str(args)))
     try:
         with start_ert_server(), ErtPluginContext() as context:
             context.plugin_manager.add_logging_handle_to_root(logging.getLogger())
+            logger.info(f"Running ert with {str(args)}")
```

Comment:
Agree with other changes, but this should stay as `.format` (logging is the exception to the rule)

Reply:
The only recommendation I've seen is to use `%s`-formatting: 
https://docs.python.org/3/howto/logging.html#logging-variable-data

Why should we prefer `.format` to f-strings?

Reply:
Was thinking of pylint W1202, but you are right, that only recommends `%s` formatting, and should be no difference between `f""` and `.format()`.